clients/benchmarks/client.cpp

```diff
@@ -113,6 +109,27 @@ struct perf_blas : rocblas_test_invalid
 {
 };
 
+template <typename T>
+struct perf_blas<T,
+                 typename std::enable_if<std::is_same<T, rocblas_float_complex>{}
+                                         || std::is_same<T, rocblas_double_complex>{}>::type>
```

Comment:
`typename std::enable_if<is_complex<T>>::type` can be used instead of `typename std::enable_if<std::is_same<T, rocblas_float_complex>{} || std::is_same<T, rocblas_double_complex>{}>::type`

Reply:
understand. thanks for catch that.